packages/connect/README.md

```diff
@@ -1,52 +1,40 @@
-# Polkadot JS Provider for Smoldot Light Client
+# @substrate/connect
 
-This is a prototype for using [@polkadot/api](https://polkadot.js.org/docs/api/start)
-with the [smoldot](https://npmjs.com/package/smoldot) WASM light client either by 
-passing chainspecs or using the extension by predefined chains (e.g. westend2, ksmcc3).
-
-ScProvider check existence of substrate extension. If it is installed and activated then
-smoldot clients of extension will be used. If not, a new smoldot client will start and
-sync with given chainspecs.
 ## Usage
```

Comment:
I think we still need a paragraph about PolkadotJS, otherwise users just land there and read the example and don't know what to do next. Either explain that they can use PolkadotJS, or remove all the examples altogether, but the middle ground doesn't really make sense.

Reply:
I agree that we should mention PolkadotJS and show examples on a high-level API like the `ScProvider` can be used to leverage `@substrate/connect`. However, while working on the landing page, I also thought that we could add a few "low-level" examples of the usage of the `@substrate/connect` API, like for instance:

```ts
import { WellKnownChain, createScClient } from '@substrate/connect';

const client = createScClient();

(async () => {
  const chain = await client.addWellKnownChain(
    WellKnownChain.polkadot,
    function jsonRpcCallback(response) {
      console.log('response', response);
      chain.remove();
    }
  );

  chain.sendJsonRpc(
    '{"jsonrpc":"2.0","id":"1","method":"system_health","params":[]}'
  );
})();
```